.buildkite/pipeline.yml

```diff
@@ -146,8 +119,8 @@ steps:
    if: *not_scheduled
    depends_on:
```

Comment:
hmm, why packaging depends on nettests? :thinking:

Reply:
It's buildkite dependencies, meaning he packaging step should only run after nettests are run to ensure they pass, I assume.